src/views/drivingLicence/check-your-details.html

```diff
@@ -0,0 +1,215 @@
+{% extends "base-form.njk" %}
+{% set hmpoPageKey = "check-your-details" %}
+{% set hmpoH1Class = "govuk-heading-l" %}
+{% from "hmpo-form/macro.njk" import hmpoForm %}
+{% from "hmpo-submit/macro.njk" import hmpoSubmit %}
+{% from "hmpo-radios/macro.njk" import hmpoRadios %}
+{% from "hmpo-warning-text/macro.njk" import hmpoWarningText %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% block mainContent %}
+
+    <h1 id="header" data-page="{{hmpoPageKey}}" class="govuk-heading-l">
+        {{ translate("pages.check-your-details.title") }}
+    </h1>
+
+    {{ hmpoWarningText({
+        text: translate("pages.check-your-details.warningText")
+    }) }}
+
+    {% call hmpoForm(ctx, {
+        attributes: {
+            onsubmit: 'window.disableFormSubmit()'
+        }
+    }) %}
+
+        {% if (values.issuedBy == "DVLA") %}
```

Comment:
This will need to be values.licenceIssuer now, now issuedBy

Reply:
Now amended to values.licenceIssuer.